app/controllers/omniauth_controller.rb

```diff
@@ -67,12 +65,8 @@ def failure_message
     "There was an error. Please try again in a few moments. If this problem persists, contact us at continuing-professional-development@digital.education.gov.uk"
   end
 
-  def registration_wizard_omniauth_step
-    :start
-  end
-
   def failed_sign_in_path
-    registration_wizard_show_path(:get_an_identity)
+    registration_wizard_show_path(:start)
```

Comment:
With the basic oauth kick off button being on the start page we no longer need the get_an_identity interstitial page as a non-JS fallback, the start page functions as that now.